frontend/src/components/LandingBody.tsx

```diff
@@ -0,0 +1,66 @@
+import { Container, Button } from 'react-bootstrap';
+import { useState, ReactNode } from 'react';
+import { LinkButton } from './LinkButton';
+import { Badges } from './Badges';
+import './LandingBody.css';
+interface ILandingBodyProps {
+
+}
+
+const LandingBody: React.FunctionComponent<ILandingBodyProps> = (props) => (
+    // <Container className='d-flex mt-5 justify-content-center flex-column align-items-center'>
+    <Container className='d-flex mt-5 justify-content-center flex-column align-items-center'>
+        <div>
+            <h1 className='text-center'><b>Get a Zeko Badge that certifies your contribution to DAOs on Discord.</b></h1>
+            <h1 className='text-center'><b>Keep your privacy intact</b></h1>
+            <LinkButton to='discord'>Claim your Zeko Badges!</LinkButton>
```

Comment:
I guess this should route to `mint` rather than `discord`. The Discord authentication takes place in /mint appartently